lib/db/schema/patch-034-035.sql

```diff
@@ -0,0 +1,87 @@
+CREATE TABLE IF NOT EXISTS unblockCodes (
+  uid BINARY(16) NOT NULL,
+  unblockCode VARCHAR(32) NOT NULL,
+  createdAt BIGINT SIGNED NOT NULL,
+  UNIQUE INDEX unblockCodes_uid_code(uid, unblockCode)
```

Comment:
Also, I wonder if we'll eventually want to prune unused tokens from this table, and hence whether we should add an index on `createdAt` so that the pruning job can find them more efficiently.  I guess we can add that in a separate migration though.